source/includes/project-initial-mappings-att.rst

```diff
@@ -0,0 +1,4 @@
+Before finalizing the creation of your project, you have the choice to 
+select the initial mappings. These options include starting with a 
+recommended MongoDB schema, which is automatically suggested by 
+Relational Migrator based on your relational database.
```

Comment:
(question) 
What is the reason you need the Initial Mappings info in the About this Task section as well? Is there something we are expecting a user to do or prepare in advance? Or is this information important for them to be able to perform the task? 

If no, let's consider deleting the About this Task section and keep the Initial Mapping just in the step.

Reply:
Nothing that the user needs to do explicitly, just calling attention to the feature before the users start a project. I think we could get away with leaving this only in the steps, do you have a strong preference @tomhollander ?